rules.org

```diff
@@ -1,14 +1,69 @@
 #+SETUPFILE: org-theme/setup/short.setup
-
-#+AUTHOR: Manoel Vilela
-#+DATE: <2018-02-10 Sat>
+#+AUTHOR: Manoel Vilela, Lucas S. Vieira
+#+DATE: <2020-09-20 dom>
 #+TITLE: Regras
 
+* Que grupo é esse?
+
+A *Common Lisp Brasil* é a principal comunidade brasileira a respeito da
+linguagem de  programação Common Lisp, onde  discute-se principalmente
+sobre  programação  nessa  linguagem,   mas  também  admite-se  outros
+assuntos relacionados (/não abuse/).
+
+* Quais são as regras?
+
+As  "regras"  a  seguir  são  linhas  gerais  para  garantir  uma  boa
+convivência na nossa comunidade de Common Lisp.
+
+Não temos  a prática de  reforçar regras restritivas. Ao  invés disso,
+esperamos que  os participantes  possam usufruir de  um bom  espaço de
+discussão onde majoritariamente discuta-se Common Lisp.
+
+Algumas sugestões de convivência estão listadas a seguir. Na  falta ou
+dubiedade de  algum tópico, esperamos  que o membro possa  confiar nas
+decisões  dos moderadores,  que  lidarão com  a  situação.  Caso  seja
+necessário, esta página poderá também ser modificada no futuro.
+
+1. Aja  com *boa-fé*. Escreva  com respeito e seja  respeitado. Ademais,
+   nunca aja como  se algum comentário tivesse a  intenção de ofender;
+   se esse for o caso, a moderação agirá para resolver a situação.
+2. Não  poste /spam/! Evite assuntos descorrelacionados de
+   Common Lisp, outros dialetos de Lisp ou Tecnologia em geral.
+3.  Somos muito receptivos a  discussões sobre outros dialetos de Lisp
+   como Scheme,  Clojure, LFE, etc.   Mas cuide  para que o  grupo não
+   "deixe de ser" um grupo de Common Lisp no processo.
+4.  Os moderadores  são  seus amigos  e tentam  agir  da melhor  forma
+   possível. Lembre-se que o grupo vai responder a você da mesma forma
+   como você agir.
+5. Aprenda e  discuta Common Lisp! Não faz sentido  você participar do
+   grupo se não vai explorar  uma das melhores linguagens que existem,
+   não é mesmo?
+6. Não há tolerância para /trolls/. Portanto, não alimente-os.
+7. Divulgações de links são  toleradas para membros engajados, mas com
+certa parcimônia.
+
+Os moderadores  zelam pelo  bem-estar da  comunidade. Caso  você cause
+algum problema,  poderá receber um  aviso (dependendo da  gravidade da
+ofensa).  Caso haja  um problema  grave ou  houver reincidência,  você
+poderá ser banido imediatamente.
+
+* Sobre banimentos automáticos
 
-1. Não seja um **cuzão**, não poste spam ou coisas totalmente
-   descorrelacionadas com Lisp, Tecnologia e Lain.
-2. Leia a primeira regra.
+Durante a pandemia global de 2020,  grupos de /trolls/ acharam que seria
+divertido redirecionar  pessoas para entrarem  em nosso grupo.  Como a
```

Comment:
não ficou claro o que a pandemia tem a ver com os trolls...

Reply:
Apenas foi o período quando a chuva de trolls teve seu início. A relação dos trolls com a pandemia é motivo de pesquisa, mas estima-se que seja "falta do que fazer".

Reply:
Não vejo muito motivo para alterar, mas vocês acham que isso poderia ser melhor escrito?

Reply:
Não vejo motivo para alterar, mas estou ainda aberto a sugestões.